src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -11,31 +11,36 @@
 import seedu.address.model.person.Name;
 import seedu.address.model.person.Person;
 import seedu.address.model.person.Phone;
+import seedu.address.model.person.Remark;
 import seedu.address.model.tag.Tag;
 
 /**
  * Contains utility methods for populating {@code AddressBook} with sample data.
  */
 public class SampleDataUtil {
+
+
+    public static final Remark EMPTY_REMARK = new Remark("");
+
     public static Person[] getSamplePersons() {
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Level("p1"),
-                new Address("Blk 30 Geylang Street 29, #06-40"),
+                new Address("Blk 30 Geylang Street 29, #06-40"), EMPTY_REMARK,
```

Comment:
Would it be better if our samples had sample remarks instead of empty? That way we could catch bugs and errors when we load a fresh sample copy of the Track-O Json and check if they output correctly

Reply:
Thanks for looking through, have updated the sample data to include a variation of remarks including EMPTY_REMARK!